From 5ebc88406f51fe36bfb03319f1d4fb5ed8888b55 Mon Sep 17 00:00:00 2001
From: rezso <rezso@rezso.net>
Date: Sun, 25 Jan 2015 17:13:08 +0100
Subject: [PATCH] Use MATE's own .directory files in matecc

First: Personal.directory, InternetAndNetwork.directory, Hardware.directory and System.directory shipped by gnome-menus, which is causes an unwanted gnome-menus dependency.
Second: because of different gettext domains, the translations are unused.
---
 shell/matecc.menu | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/shell/matecc.menu b/shell/matecc.menu
index b0beba5..a99fdf7 100644
--- a/shell/matecc.menu
+++ b/shell/matecc.menu
@@ -46,7 +46,7 @@
   <!-- Personal category -->
   <Menu>
     <Name>Personal</Name>
-    <Directory>Personal.directory</Directory>
+    <Directory>mate-personal.directory</Directory>
     <Include>
       <And>
         <Category>Settings</Category>
@@ -62,7 +62,7 @@
   <!-- Internet and Network category -->
   <Menu>
     <Name>Internet and Network</Name>
-    <Directory>InternetAndNetwork.directory</Directory>
+    <Directory>mate-internet-and-network.directory</Directory>
     <Include>
       <And>
         <Category>Settings</Category>
@@ -74,7 +74,7 @@
   <!-- Hardware category -->
   <Menu>
     <Name>Hardware</Name>
-    <Directory>Hardware.directory</Directory>
+    <Directory>mate-hardware.directory</Directory>
     <Include>
       <And>
         <Category>Settings</Category>
@@ -86,7 +86,7 @@
   <!-- System category -->
   <Menu>
     <Name>System</Name>
-    <Directory>System.directory</Directory>
+    <Directory>mate-system.directory</Directory>
     <Include>
       <And>
         <Category>Settings</Category>

